src/Microsoft.Identity.Client/ApiConfig/Executors/PublicClientExecutor.cs

```diff
@@ -125,9 +125,9 @@ public PublicClientExecutor(IServiceBundle serviceBundle, PublicClientApplicatio
 
             var coreUiParent = interactiveParameters.UiParent;
 
-#if ANDROID || iOS
-            coreUiParent.UseEmbeddedWebview = interactiveParameters.UseEmbeddedWebView;
-#endif
+            coreUiParent.UseEmbeddedWebview = interactiveParameters.UseEmbeddedWebView.GetValueOrDefault(
```

Comment:
makes sense, however still seems to be other ways to do the same without introducing Maybe.

Reply:
Yes, after looking at this again I decided to replace it with an enum "NotSpecified", "Embedded", "System".